libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/helpers/VcHelper.java

```diff
@@ -77,6 +75,36 @@ public static boolean isSuccessfulVcs(List<SignedJWT> vcs) throws ParseException
         return true;
     }
 
+    public static List<VcStoreItem> filterVCBasedOnProfileType(
+            List<VcStoreItem> vcStoreItems, ProfileType profileType) {
+        List<VcStoreItem> filteredVCs;
+        if (profileType.equals(ProfileType.GPG45)) {
+            filteredVCs =
+                    vcStoreItems.stream()
+                            .filter(
+                                    vcItem ->
+                                            !OPERATIONAL_CRIS.contains(
+                                                    vcItem.getCredentialIssuer()))
+                            .toList();
+        } else {
+            filteredVCs =
+                    vcStoreItems.stream()
+                            .filter(
+                                    vcItem ->
+                                            (OPERATIONAL_CRIS.contains(vcItem.getCredentialIssuer())
+                                                    || vcItem.getCredentialIssuer()
+                                                            .equals(TICF_CRI)))
+                            .toList();
+        }
+        return filteredVCs;
```

Comment:
(can we just put the return inside the if to eliminate this variable?)

Reply:
multiple returns frm method gives code smell.